src/templates/functions.jsx

```diff
@@ -2,16 +2,96 @@ import React from 'react';
 
 import { html as toHtml } from '../indices-to-html';
 
+/**
+ * Adds markup to some common text patterns
+ *
+ *  - Bullet lists
+ *  - Todo lists, WP style
+ *  - Inline `code` snippets
+ *  - Header: description/explanation paragraphs
+ *
+ * Note: This code is only meant to serve until a
+ * proper parser can be built up to convert the
+ * unstructured text into structured data. Since at
+ * this time we still create HTML strings directly
+ * and on every render this function will serve
+ * sufficiently but it should not be looked upon
+ * as good example code!
+ *
+ * @param {String} text input list of blocks as HTML string
+ * @returns {String} marked-up text
+ */
+const toBlocks = text =>
+    text.split('\n').reduce(({ out, inList }, raw) => {
+        // emphasized definition-like text
+        // Some value: some description
+        // Header: Value
+        //
+        // Not! This is fun. This: again; isn't emphasized.
+        // May detect false positive if colon found in first sentence.
+        const defined = /^[\w\s-_]+:/.test(raw)
+            ? `<strong>${raw.split(':')[0]}:</strong>${raw.replace(/^[^:]:+/, '')}`
+            : raw;
+
+        // inline `code` snippets
+        const coded = defined.replace(/`([^`]+)`/, '<code>$1</code>');
```

Comment:
@gwwar I absolutely want to do this properly. it's basically only because I'm tired of seeing such unstyled content that I did this as a "meh, it's much better than it was"

I think you are right though in that inline code should have precedence over a list and I'll make this change.

having had such major issues in the past held me back from making some of these smaller improvements. when we "do it properly" (and why I hesitate from just doing that now) I want to make a drastic change in how we do all of the transformation. right now we pump API responses directly into the app and transform the data on every render. the approach I took in [dmsnell/wrnc](https://github.com/dmsnell/wrnc) was to [transform the API data when it comes into the system](https://github.com/dmsnell/wrnc/blob/58fe073c1c0b196ee46d02eb8dc178fbdc3c4227/src/api-poller/from-api.js) then we never have to repeat the transformation again. much faster, much saner. (data models, who would'a thunk it?)